src/main/java/com/agnieszkapawska/flashcards/domain/dtos/FlashcardResponseDto.java

```diff
@@ -0,0 +1,12 @@
+package com.agnieszkapawska.flashcards.domain.dtos;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class FlashcardResponseDto {
```

Comment:
```suggestion
public class FlashcardSaveResponseDto {
```
I guess there will be also response for updating and deleting. Do you want to use the same response or you will have separate classes for all cases? I'm not sure what is better option, just asking :)